applications/launchpad_v2/src/components/Box/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { StyledBox } from './styles'
+
+import { BoxProps } from './types'
+
+const Box = ({ children, gradient, border, style: inlineStyle }: BoxProps) => {
+  const style = {
+    border: border === false ? 'none' : undefined,
+    background:
+      gradient &&
+      `
+      linear-gradient(
+      45deg,
```

Comment:
NIT: Maybe the `45deg` could be also parametrised with props?

Reply:
added optional `rotation` to gradient, defaulting to 45